### ISSUE TITLE
error output in browserstack test run shows it's using outdated Cypress version of 4.12

### ISSUE THREAD

**Issue by bdefore** [browserstack/browserstack-cypress-cli#75]
Is this expected? I'm using features in version 5, specifically 5.3.0
```
TimeoutError: operation timed out
    at afterTimeout (/root/.cache/Cypress/4.12.1/Cypress/resources/app/packages/server/node_modules/bluebird/js/release/timers.js:46:19)
    at Timeout.timeoutTimeout [as _onTimeout] (/root/.cache/Cypress/4.12.1/Cypress/resources/app/packages/server/node_modules/bluebird/js/release/timers.js:76:13)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
```

**Comment by Namrata-m13** [comment]
@bdefore Do you encounter the same error when executing the tests on your system using version 4.12?
Also, a build-id/session-id for the shared error will be quite helpful in further debugging.

**Comment by bdefore** [comment]
i haven't tried to run on 4.12 locally, but here is a build id:
494333442ccccced7478f87cba49c7b6dd1c4931
and session id: 9e8f3a0367af246050ea35e3fbf55418d6a94792. i see the output
of a 4.12 version in the console logs towards the end

**Comment by Namrata-m13** [comment]
@bdefore To debug this further, could you please try the below-mentioned steps-
1. Try running the specs on Cypress v4.x locally. Preferred 4.12
2. The test which is trying to load a file is not inside the test directory. Could you please move that file to within the same level or level down the browserstack.json?

**Comment by karanshah-browserstack** [comment]
Hi, This should be fixed in v1.6.0 which we released yesterday. Please feel free to reach out to us if you still feel any issues .
